grails-app/views/speciesListItem/list.gsp

```diff
@@ -142,7 +142,15 @@
                                 row.find("td.imageUrl").html("");
                             } else {
                                 var bie = "<a href=${bieUrl}/species/" + guid + " title='click to view species page'>"+result.matchedName + "</a>"
-                                row.find("td.matchedName").html(bie)
+
+                                if (result?.matchedSpecies?.vernacularName)
+                                    bie +="<br/>" + result?.matchedSpecies.vernacularName
+                                if (result?.matchedSpecies?.scientificNameAuthorship)
+                                    bie += "<br/>Autheor(s):" + result?.matchedSpecies.scientificNameAuthorship
```

Comment:
Should be "Author(s)" and preferably i18n'd

Reply:
i18n added

Reply:
I'm still reading "Autheor(s)" rather than "Author(s)"?